docs/articles/remoting/transports.md

```diff
@@ -155,5 +161,27 @@ Why is this distinction important? Why do we care about registering an publicly
 
 By default, Akka.Remote assumes that `hostname` is publicly accessible and will use that as the `public-hostname` value. But in the even that it's not AND some of your Akka.NET applications might need to contact this process then you need to set a publicly accessible hostname. 
 
+## Running Mixed Version of Akka.NET
+As of v1.4.20, the primitive serializer wire format in Akka.Remote have been changed to improve
+serialized message size. This change introduces a bug that causes remote communication problem 
+between systems with v1.4.20 and above and systems with v1.4.19 and below. However, this bug was not 
+found until v1.4.25, which means that __v1.4.20 - v1.4.25 can not be mixed with any older 
+versions of Akka.NET__. In order for a new system to communicate with systems with v1.4.19 or below, 
+you will have to use v1.4.26 or above.
+
+In order for a v1.4.26 and above system to communicate properly with v1.4.19 and below systems, 
+you will have to  add this into the HOCON settings:
+```
+akka.actor.serialization-settings.primitive.use-neutral-primitives = true
+```
+This will revert the new serializer to the old behaviour and allows them to be completely compatible
+with older versions of Akka.NET.
+
+### Rolling an Update From v1.4.19 to v1.4.26 
```

Comment:
This oversells the problem, IMHO - the issue only occurs when string, int, etc are sent directly over the wire without a wrapper message. It's been reported once in the several months since v1.4.20 has been out.